tests/system_tests/test_core/test_public_api.py

```diff
@@ -487,6 +487,101 @@ def test_projects(user, wandb_backend_spy):
     assert sum([1 for _ in projects]) == 2
 
 
+def test_delete_files_for_multiple_runs(
+    user,
+    wandb_backend_spy,
+):
+    gql = wandb_backend_spy.gql
+    body = {
+        "data": {
+            "project": {
+                "runCount": 2,
+                "runs": {
+                    "edges": [
+                        {
+                            "node": {
+                                "name": "test",
+                                "id": "test",
+                                "sweep_name": None,
+                            },
+                        },
+                        {
+                            "node": {
+                                "name": "test",
+                                "id": "test2",
+                                "sweep_name": None,
+                            },
+                        },
+                    ],
+                    "pageInfo": {
+                        "endCursor": None,
+                        "hasNextPage": False,
+                    },
+                },
+            },
+        },
+    }
```

Comment:
Could you factor out the other response bodies and define them at the top here as well? There's a lot of data here which makes it difficult to see what the test is actually doing.